packages/salesforcedx-vscode-core/package.json

```diff
@@ -181,6 +182,10 @@
         {
           "command": "sfdx.force.debugger.stop",
           "when": "sfdx:project_opened"
+        },
+        {
+          "command": "refreshsobjects",
```

Comment:
Put this under the sfdx.force.internal.refreshsobjects namespace for now.
Command ids are meant to be unique so it's best to qualify them properly with some prefix.

Reply:
OK, will do - I wasn't sure whether the pattern was to only use sfdx.force for items that were CLI related to sfdx....